test/bin/mdx-test/expect/empty-lines/test-case.md

```diff
@@ -5,6 +5,10 @@ This shell block contains an empty line within a padded block:
 
 ```
 
+```sh
+  $ echo "Hello...\n\nworld!"
```

Comment:
This doesn't work with my shell for instance.

In my version of this PR I added the following to `test/bin/mdx-test/expected/padding/test-case.md`:
````markdown
Paddign shouldn't be added to empty lines:
```sh
  $ echo "" && echo foo

  foo
```
````

Could you try it to see if it fixes the CI?

Reply:
done!